lib/src/presentation/view/onboarding/splash/splash_view.dart

```diff
@@ -42,7 +42,7 @@ class _SplashViewState extends State<SplashView>
       
       listener: (context, state) {
         print(" error is ${state.errorMessage}");
-        if (state.isCheckAuth== false) {
+        if (state.isVrifyingOtp== false) {
```

Comment:
Why verifying otp flag for checking auth. Please make sure to understand the flow of state managament for checking auth.